lib/pages/timetable/view/timetable_page.dart

```diff
@@ -125,6 +125,14 @@ class _TimetablePageState extends State<TimetablePage> {
           ),
         ],
       ),
+      floatingActionButton: FloatingActionButton(
```

Comment:
FABs are generaly not recommended: https://www.reddit.com/r/userexperience/comments/38cbqq/material_design_why_the_floating_action_button_is/

If we do end up adding a FAB on this page, it'll be for adding an event. This way it's super confusing, as a user I don't understand immediately what this button does and it opens a new browser window asking for permissions. Kinda sketchy.

For now, I'd say just add another item on the settings page that does this.